packages/corvid-local-site/src/siteConverter.js

```diff
@@ -199,13 +201,19 @@ const editorDocumentToLocalDocumentFiles = siteDocument => {
 };
 
 const localDocumentFilesToEditorDocument = localDocumentFiles => {
-  const parsedLocalDocumentFiles = localDocumentFiles.map(file =>
-    isPathOfWixFile(file.path)
-      ? Object.assign({}, file, {
+  const parsedLocalDocumentFiles = localDocumentFiles.map(file => {
+    if (isPathOfWixFile(file.path)) {
+      try {
+        return Object.assign({}, file, {
           content: fromWixFileContent(file.content).content
-        })
-      : file
-  );
+        });
+      } catch (_) {
```

Comment:
we shouldn't ignore the actual original error

Reply:
done